Userland/Applications/PixelPaint/Tools/CloneTool.cpp

```diff
@@ -61,6 +61,7 @@ Variant<Gfx::StandardCursor, NonnullRefPtr<Gfx::Bitmap>> CloneTool::cursor()
 
 void CloneTool::on_mousemove(Layer* layer, MouseEvent& event)
 {
+    set_alt_key_override();
```

Comment:
Why set this at every mouse{move, down}?
You can just set it in the constructor right?